### ISSUE TITLE
AttributeError raised on Search.entries

### ISSUE THREAD

**Issue by cparker-zen** [salvoventura/pypexels#7]
example usage/query:
```
images = list()
search_results = py_pexels.search(query='tr', per_page=20)
for photo in search_results.entries:
      ...
```
results in:
```
  File "/app/services.py", line 79, in get_search_images
    for photo in search_results.entries:
  File "/usr/local/lib/python3.6/site-packages/pypexels/src/search.py", line 37, in entries
    for entry in self.body.get('photos', []):
AttributeError: 'NoneType' object has no attribute 'get'
```

It's intermittent from what I can tell. It's likely the API request failed which resulted in the `Search.body` property being `None`.

**Comment by salvoventura** [comment]
Quite interesting. I am thinking to raise an exception for network errors and let the user deal with that. Open to ideas, but I think the common approach for libraries is to bubble up core issues so that the user is aware of them. Like in this case, but with little more context.

**Comment by cparker-zen** [comment]
Sounds good to me!

**Comment by salvoventura** [comment]
I put the fix in release 1.0.0b4
